src/editor.py

```diff
@@ -418,7 +418,8 @@ def generate_scoring_plays(self,files):
             return scoringplays
 
 
-    def generate_highlights(self,files):
+    def generate_highlights(self,files,myteam="",theater_link=False):
+        import datetime

```

Comment:
This is already imported at the top of the file [here](https://github.com/toddrob99/Baseball-GDT-Bot/blob/6f0b9c5dafdf617379369155a684d9120c46f6c8/src/editor.py#L8)